src/components/common/SummaryDirectors.vue

```diff
@@ -67,17 +87,29 @@
                     <base-address :address="director.deliveryAddress" />
                   </div>
 
-                  <div class="address same-address" v-if="isBenBcCccUlc">
+                  <div
+                    v-if="isBenBcCccUlc"
+                    class="address same-address"
+                  >
                     <span v-if="isSame(director.deliveryAddress, director.mailingAddress)">
                       Same as Delivery Address
                     </span>
-                    <base-address v-else :address="director.mailingAddress" />
+                    <base-address
```

Comment:
Could you rename this to `<BaseAddress` please? Also search for others in this file and globally.

Reply:
Thanks.